OracleWebCenterContent/README.md

```diff
@@ -0,0 +1,16 @@
+# Oracle WebCenter Content Container Image
```

Comment:
"container" and "image" should be all lowercase. They are not part of the product name nor are they proper nouns.